test/config-next/va.json

```diff
@@ -22,7 +22,8 @@
       "address": ":9092",
       "clientNames": [
         "health-checker.service.consul",
-        "ra.service.consul"
+        "ra.service.consul",
+        "wrong.service.consul"
```

Comment:
I see you're adding `wrong.service.consul` in some of the client names, but your change only affects validation of server certificates. The `clientNames` fields should be unaffected.

Reply:
I believed this to be the case at first as well. However, the 'wrong' certificate now presented by the RA, which we're accounting for with the override, is the same that the RA uses for its own client connections.

Reply:
Ah good point.

What do you think of this: rather than have a single service where the naming scheme is "wrong" (i.e. doesn't match the `<service>.<domain>` pattern), why not have two whole separate naming schemes: one for the certificates (e.g. `ra.boulder` as we had before, or anything really) and one for service discovery (`ra.service.consul`). Then we would have all of the services use this naming scheme consistently for their HostOverride / ClientNames, and also take the opportunity to make everything in config-next consistently use SRVLookup. Right now I feel like we're winding up in a combinatorial explosion of things we want to test together: config-next with serverAddress; config-next with SRVLookup; config-next with SRVLookup and HostOverride.

`serverAddress` is already tested in test/config; config-next is about the future, so it can use SRVLookup consistently. And I think the plan is that anywhere we deploy SRVLookup, we will also be deploying HostOverride, so it makes sense to have all of our config-next instances use both.

Reply:
Ended up really digging out clean this turned out. Thanks again.